Adyen/httpclient.py

```diff
@@ -89,22 +89,10 @@ def _pycurl_post(self,
 
         response_headers = {}
 
-        def handle_header(header_line):
-            header_line = header_line.decode('iso-8859-1')
-            if ':' in header_line:
-                name, value = header_line.split(':', 1)
-                name = name.strip()
-                value = value.strip()
-                response_headers[name] = value
-
         curl = pycurl.Curl()
         curl.setopt(curl.URL, url)
 
-        if sys.version_info[0] >= 3:
-            stringbuffer = BytesIO()
-        else:
-            stringbuffer = StringIO()
-        # stringbuffer = StringIO()
+        stringbuffer = StringIO()
```

Comment:
Hi @thaelathy ,
I receive a "TypeError: cannot use a string pattern on a bytes-like object" when using Python3 with the pycurl package, do you have a solution to make it compatible with both Python 2 and 3?

Thanks,
Alessio
Adyen

Reply:
Can I get this error in unit tests? Mb you use python 3.5?